pkg/ctl/utils/describe_addon_versions.go

```diff
@@ -44,7 +44,7 @@ func describeAddonVersionsCmd(cmd *cmdutils.Cmd) {
 }
 
 func describeAddonVersions(cmd *cmdutils.Cmd, addonName, k8sVersion, clusterName string) error {
-	clusterProvider, err := cmd.NewProviderForExistingCluster()
+	clusterProvider, err := cmd.NewCtl()
```

Comment:
This should fix the integration tests but we won't get a user-friendly error message if it's a non-EKS cluster. I think we can make this improvement later in a patch release.